docs/02-app/01-building-your-application/08-testing/01-vitest.mdx

```diff
@@ -21,15 +21,17 @@ npx create-next-app@latest --example with-vitest with-vitest-app
 To manually set up Vitest, install `vitest` and the following packages as dev dependencies:
 
 ```bash filename="Terminal"
-npm install -D vitest @vitejs/plugin-react jsdom @testing-library/react
+npm install -D vitest @vitejs/plugin-react jsdom @testing-library/react @testing-library/dom
 # or
-yarn add -D vitest @vitejs/plugin-react jsdom @testing-library/react
+yarn add -D vitest @vitejs/plugin-react jsdom @testing-library/react @testing-library/dom
 # or
-pnpm install -D vitest @vitejs/plugin-react jsdom @testing-library/react
+pnpm install -D vitest @vitejs/plugin-react jsdom @testing-library/react @testing-library/dom
 # or
-bun add -D vitest @vitejs/plugin-react jsdom @testing-library/react
+bun add -D vitest @vitejs/plugin-react jsdom @testing-library/react @testing-library/dom
 ```
 
+Beginning from [React Testing Library 16](https://github.com/testing-library/react-testing-library/releases/tag/v16.0.0), `@testing-library/dom` is a peer dependency and needs to be explicitly installed.
```

Comment:
For further readers, I'm expecting them to be installing above 16, do we still need this explanation?

Reply:
Done